modules/programs/rcm.nix

```diff
@@ -0,0 +1,36 @@
+{ config, pkgs, user, lib, ... }:
+
+with lib;
+with lib.strings;
+let
+  cfg = config.programs.rcm;
+
+  stringifyList = l:
+    if isList l then ''"${concatStringsSep " " l}"'' else stringifyList [ l ];
```

Comment:
`toString <list>` already does that.

Reply:
Ah sorry, I had missed the `"` wrapping.

I think at that point you could just inline it into `mkKeyValue`.

Reply:
I'd rather inline only the branching part for readability and less repeated quote confusion.